vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/main/java/com/vaadin/flow/component/combobox/test/MultiSelectComboBoxDetachReattachPage.java

```diff
@@ -0,0 +1,29 @@
+package com.vaadin.flow.component.combobox.test;
+
+import com.vaadin.flow.component.combobox.MultiSelectComboBox;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.NativeButton;
+import com.vaadin.flow.router.Route;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+@Route("vaadin-multi-select-combo-box/detach-reattach")
+public class MultiSelectComboBoxDetachReattachPage extends Div {
```

Comment:
Multi-select combo box is not affected currently, as it uses the `selectedItems` property as field value, which is automatically synced by the abstract field class. Still can't hurt to add a test.